.github/workflows/macos.yml

```diff
@@ -102,8 +102,8 @@ jobs:
         python3 -m pip install --upgrade pip
         python3 -m pip install cython coverage
     - env:
-        CPPFLAGS: "-I/opt/homebrew/include"
-        LDFLAGS: "-L/opt/homebrew/lib"
+        CPPFLAGS: "-I/opt/homebrew/opt/include"
+        LDFLAGS: "-L/opt/homebrew/opt/lib"
         MESON_ARGS: --unity=${{ matrix.unity }}
```

Comment:
Is this change in the right commit? Why isn't it done in unittests-appleclang? Why is homebrew using a double /opt anyway...

Reply:
grrrrrrrrrrr. that rebased *back in*!